Gemfile.lock

```diff
@@ -0,0 +1,200 @@
+GEM
```

Comment:
This seems like kindof a whole lot to install. All I needed was `jekyll` and `jekyll-paginate` to get it working. I see several themes we don't need as well. Will this help or just be a ton to download?